src/app/components/project-selector/projectSelector.component.js

```diff
@@ -49,6 +51,15 @@ class ProjectSelectorController {
     this.onChange({ project });
     this.isOpen = false;
   }
+
+  signIn() {
+    this.$mdDialog.cancel(
+      this.$state.target('login', {
+        nextState: this.$state.nextState,
+        nextStateParams: this.$state.nextStateParams,
```

Comment:
are `nextState` and `nextStateParams` ever set? You are setting them again in `projectView.hook.js`